Source/Scene/ModelExperimental/GeometryPipelineStage.js

```diff
@@ -31,7 +31,7 @@ var GeometryPipelineStage = {};
  * @private
  */
 GeometryPipelineStage.process = function (renderResources, primitive) {
-  var attributeIndex = 0;
+  var attributeIndex = renderResources.attributeIndex;
```

Comment:
Oh I see now, this is because instancing might introduce some attributes before we even get here. Could you put a comment explaining this?